Replace metricsmap-bpf-prom-sync with Prometheus Collector pattern (#27370)
cilium-team/cilium: Documentation/community/governance/commit_access.rst

Code of Conduct
~~~~~~~~~~~~~~~

> `Code of Conduct`_ violations by community members will be discussed and

Reviewer: Use present tense
```suggestion
`Code of Conduct`_ violations by community members are discussed and
```

Author: Not part of this PR.

issue will be resolved by voting. The voting process is a simple majority in
which each committer and each maintainer receives one vote.

> Votes are done in the Slack channel ``#committers`` using Slack polls. A
> failure to vote is an implicit abstention.

Reviewer: I think we can be gentler.
```suggestion
Votes are done in the Slack channel ``#committers`` using Slack polls.
Declining to vote is an implicit abstention.
```

Author: Not part of this PR.
